rollup.cssBundler.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs')
```

Comment:
I personally would prefer one of the alternatives I mentioned to maintaining our own custom Rollup plugin but this is fine for merging (and a neat reference).